R/fiveyeargwsum-styles.R

```diff
@@ -1,12 +1,20 @@
-getFiveyearStyle <- function(data, info=NULL) {
+getFiveyearStyle <- function(data, info=NULL, invertedPlot = FALSE) {
 
     x <- data[[1]]$time
     y <- data[[1]]$value
+    
+    #semantics for min/max are swapped on inverted plots
+    minMaxLabels = c("Max. Instantaneous", "Min. Instantaneous");
+    if(invertedPlot) {
+      minMaxLabels = c("Min. Instantaneous", "Max. Instantaneous");
+    }
+    
```

Comment:
I guess that's just where I've been changing it so far, but yeah it's not really data munging bc it's the title https://github.com/USGS-R/repgen/blob/master/R/dvhydrograph-data.R#L92-L112